zuul.d/edpm_multinode.yaml

```diff
@@ -457,13 +399,13 @@
             networks:
               default: {}
               internal-api:
-                trunk_parent: default
+                trunk-parent: default
                 skip-nm-configuration: true
               tenant:
-                trunk_parent: default
+                trunk-parent: default
                 skip-nm-configuration: true
               storage:
-                trunk_parent: default
+                trunk-parent: default
                 skip-nm-configuration: true
         instances:
```

Comment:
It doesnt nee the compute-0 anymore - it will get the compute info from the group-templates?

By compute-0 I mean:
```          compute-0:
            skip-nm-configuration: true
            networks:
              default:
                ip: "192.168.122.100"
              internal-api:
                ip: "172.17.0.100"
                trunk_parent: default
              storage:
                ip: "172.18.0.100"
                trunk_parent: default
              tenant:
                ip: "172.19.0.100"
                trunk_parent: default```

Reply:
Yes that's correct but I will request that @pablintino confirm.

Reply:
Yes, that's the idea. In general, if there's a group for instances, you may want to use the templates.

Reply:
ack, thanks